test_scripts/API/CancelInteraction/001_SendCancelInteraction_PerformInteraction_HMIReceivesRequest.lua

```diff
@@ -38,10 +37,32 @@ local rpcInteraction = {
   hmi_name = "UI.PerformInteraction",
   hmi_name2 = "VR.PerformInteraction",
   params = {
+    cancelID = 99,
     initialText = "hello",
     interactionMode = 0,
     interactionChoiceSetIDList = { 42 },
     helpPrompt = { { text = "tts_chunk", type = "SILENCE" } }
+  },
+  ui_params = {
+    cancelID = 99,
+    choiceSet = {
+      { choiceID = 1, menuName = "choice1" },
+      { choiceID = 2, menuName = "choice2" }
+    },
+    initialText = {
+      fieldName = "initialInteractionText",
+      fieldText = "hello"
+    },
+    timeout = 10000
+  },
+  vr_params = {
+    helpPrompt = {
```

Comment:
Add `cancelID = 99`

Reply:
added in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2206/commits/085ecf3126b192f8562f75bf9b3718c6cdde3111